### ISSUE TITLE
VolumeReconstructor.exe error?

### ISSUE THREAD

**Issue by chrissMD** [PlusToolkit/PlusLib#685]
[TrackedImageSequence_20191207_172327_config.txt](https://github.com/PlusToolkit/PlusLib/files/4574111/TrackedImageSequence_20191207_172327_config.txt)

After entering the command line as in explained here: [link](http://perk-software.cs.queensu.ca/plus/doc/nightly/user/ApplicationVolumeReconstructor.html), I added the name of my config file and the name of already recorded sequence. Beforehand, I added the config file in PLUS CONFIG folder and my data file in the Plus DATA folder. 

I Used as config file the same config file that I used for data acquisition!

When running I get this error:
`|ERROR|000.016000| Reading device set configuration file failed: TrackedImageSequence_20191207_172327_config.xml--source-seq-file=TrackedImageSequence_20191207_172327_ICE_3D_Cubes_Intervall_0_1007.nrrd--output-volume-file=vtkPlusVolumeReconstructorWithHoleFillingTest1Output_pigs.mha--image-to-reference-transform=ImageToReference does not exist in the current directory or in D:/pigs_december/Converted_Data| in e:\d\ptnp64b\pluslib\src\pluscommon\PlusXmlUtils.h(42)
|ERROR|000.028000| Unable to read configuration from file TrackedImageSequence_20191207_172327_config.xml--source-seq-file=TrackedImageSequence_20191207_172327_ICE_3D_Cubes_Intervall_0_1007.nrrd--output-volume-file=vtkPlusVolumeReconstructorWithHoleFillingTest1Output_pigs.mha--image-to-reference-transform=ImageToReference| in E:\D\PTNP64b\PlusLib\src\PlusVolumeReconstruction\Tools\VolumeReconstructor.cxx(116)`

![Vol_rec_error](https://user-images.githubusercontent.com/48135643/80956216-12381c00-8e01-11ea-869f-3fdb7b5ee0e9.PNG)

I Used as config file the same config file that I used for data acquisition!

**Comment by chrissMD** [comment]
Update of the error!
![Vol_rec_error1](https://user-images.githubusercontent.com/48135643/80958291-0e0dfd80-8e05-11ea-9469-7635b176d5ba.PNG)

**Comment by chrissMD** [comment]
Update 2! I changed my config file (attached) and renamed my source file. Now I get e new error. The program does not find the source file. Also, the application changed by itself the directory where is searching for data from the "data" folder of Plus to another folder. I added the source file in both directories, the "data" folder in Plus and in the new folder where it changed

[PlusConfiguration_mysequence.txt](https://github.com/PlusToolkit/PlusLib/files/4574336/PlusConfiguration_mysequence.txt)


![Vol_rec_error2](https://user-images.githubusercontent.com/48135643/80961334-06e9ee00-8e0b-11ea-8b32-9204c22e8ed8.PNG)
